orbit/pkg/useraction/mdm_migration_darwin.go

```diff
@@ -56,15 +61,23 @@ var mdmMigrationTemplatePreSonoma = template.Must(template.New("mdmMigrationTemp
 
 Select **Start** and look for this notification in your notification center:` +
 	"\n\n![Image showing MDM migration notification](https://fleetdm.com/images/permanent/mdm-migration-screenshot-notification-2048x480.png)\n\n" +
-	"After you start, this window will popup every 15-20 minutes until you finish.",
+	"After you start, this window will popup every 3 minutes until you finish.",
+))
+
+var mdmManualMigrationTemplate = template.Must(template.New("").Parse(`
+## Migrate to Fleet
+
+Select **Start** and My device page will appear soon:` +
+	"\n\n![Image showing MDM migration notification](https://fleetdm.com/images/permanent/mdm-manual-migration-1024x500.png)\n\n" +
+	"After you start, this window will popup every 3 minutes until you finish.",
```

Comment:
⚠️ important: windows makes more sense, but heads up that the design says "dialog" for manual https://www.figma.com/design/B9y0KXmF8Az0Ggp7FdHqxr/%2319625-Improve-migration-experience-on-macOS-Sonoma?node-id=1355-10152&t=eY51NDSlFJQdhVS1-4

```suggestion
	"After you start, this dialog will popup every 3 minutes until you finish.",
```